schemas/bridging/findInstancesAgentRequest.schema.json

```diff
@@ -42,7 +42,14 @@
               "$ref": "common.schema.json#/$defs/AgentDestination"
             },
             "source": {
-              "$ref": "common.schema.json#/$defs/AppRequestSource"
+              "oneOf": [
```

Comment:
Watch out for uses of `oneOf` as it requires that the input must match exactly one of the schemas. If the schemas overlap it can't be satisfied in some cases. Hence, `anyOf` is often safer.

However, this should be ok here as the required and optional fields differ for these types.